### PR TITLE
[zh] Add "WebCodecs API" translation

### PR DESCRIPTION
**Pull request by gxy5202**
Add "WebCodecs API" simplified Chinese translation.

### Description

Add "WebCodecs API" simplified Chinese translation.

### Motivation

There is no zh-CN version for "WebCodecs API".

### Additional details

原文 https://developer.mozilla.org/en-US/docs/Web/API/WebCodecs_API

### CODE REVIEW

File: files/zh-cn/web/api/webcodecs_api/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: WebCodecs API
+slug: Web/API/WebCodecs_API
+---
+
+{{DefaultAPISidebar("WebCodecs API")}}
+
+**WebCodecs API**为 web 开发者提供了对视频流的单个帧和音频数据块的底层访问能力。
+这对于那些需要完全控制媒体处理方式的web应用程序非常有用。
+例如，视频或音频编辑器，以及视频会议。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/webcodecs_api/index.md:10]:
```suggestion
**WebCodecs API**为 web 开发者提供了对视频流的单个帧和音频数据块的底层访问能力。这对于那些需要完全控制媒体处理方式的web应用程序非常有用。例如，视频或音频编辑器，以及视频会议。
```

File: files/zh-cn/web/api/webcodecs_api/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## 概念和用法
+
+许多 Web API 在内部都使用了媒体编码器。
+例如， {{domxref("Web Audio API")}}, 以及 {{domxref("WebRTC API")}}。
+然而，这些 API 不允许开发者处理视频流的单个帧和未合成的编码音频块或视频块。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/webcodecs_api/index.md:16]:
```suggestion
许多 Web API 在内部都使用了媒体编码器。例如，{{domxref("Web Audio API")}}，以及 {{domxref("WebRTC API")}}。然而，这些 API 不允许开发者处理视频流的单个帧和未合成的编码音频块或视频块。
```

File: files/zh-cn/web/api/webcodecs_api/index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+Web 开发者通常使用 WebAssembly 来绕过这一限制，并在浏览器中使用媒体编解码器。然而，这需要额外的带宽来下载浏览器中已经存在的编解码器，降低了性能和电源效率，并增加了额外的开发成本。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/webcodecs_api/index.md:18]:
```suggestion
Web 开发者通常使用 WebAssembly 来绕过这一限制，并在浏览器中使用媒体编解码器。然而，这需要额外的带宽来下载浏览器中已经存在的编解码器，降低了性能和能效，并增加了额外的开发成本。
```

File: files/zh-cn/web/api/webcodecs_api/index.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+WebCodecs API 提供了对浏览器中已存在的编解码器的访问能力。
+它可以访问原始视频帧、音频数据块、图像解码器、音频和视频的编码器及解码器
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/webcodecs_api/index.md:21]:
```suggestion
WebCodecs API 提供了对浏览器中已存在的编解码器的访问能力。它可以访问原始视频帧、音频数据块、图像解码器、音频和视频的编码器及解码器
```

File: files/zh-cn/web/api/webcodecs_api/index.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  - : 解包和解码图像数据，允许访问动画图像中的帧序列。
+- {{domxref("ImageTrackList")}}
+  - : 表示图像中可用的轨道列表。
+- {{domxref("ImageTrack")}}
+  - : 表示单个图像轨道。
+
+## 示例
+
+在下面这个例子中，获取从 {{domxref("MediaStreamTrackProcessor")}} 中返回的帧，然后对他们进行编码。
+查看完整示例并在文章 [Video processing with WebCodecs](https://web.dev/webcodecs/) 中了解更多。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/webcodecs_api/index.md:53]:
```suggestion
在下面这个例子中，获取从 {{domxref("MediaStreamTrackProcessor")}} 中返回的帧，然后对他们进行编码。在文章 [Video processing with WebCodecs](https://web.dev/webcodecs/) 查看完整示例并了解更多细节。
```

**Review by JasonLamv-t — COMMENTED** [review]
Some suggestions

File: files/zh-cn/web/api/webcodecs_api/index.md (hunk 1 above)
Comment (fwqaaq) [inline files/zh-cn/web/api/webcodecs_api/index.md:10]:
```suggestion
**WebCodecs API** 为 web 开发者提供了对视频流的单个帧和音频数据块的底层访问能力。这对于那些需要完全控制媒体处理方式的 web 应用程序非常有用。例如，视频或音频编辑器，以及视频会议。
```

参考：https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**